javascript/utils/index.ts

```diff
@@ -84,11 +81,13 @@ export function runNativeCommand<ReturnType = NativeArg>(
   }
 
   if (isAndroid()) {
-    return NativeModules.UIManager.dispatchViewManagerCommand(
+    UIManager.dispatchViewManagerCommand(
       handle,
       managerInstance.Commands[name],
       args,
     );
+
+    return null as ReturnType;
```

Comment:
It's not nice, but will probably get rid of it when migrating to the new architecture. Only other possibility I see currently is to split the `runNativeCommand` for iOS and Android.

Reply:
Why do we need to return null instead of `NativeModules.UIManager`?

Reply:
It's a void function, and when changing the return type this will break types withing `usenativeBridge`. We can only separate by platform, because iOS expects the return, while Android uses a callback.

![Bildschirmfoto 2024-11-22 um 10 14 05](https://github.com/user-attachments/assets/bfe60ae4-e9b6-4e5c-978a-84bbbe143cec)

Reply:
I see. What do you think about adding short explanation as a comment?

Reply:
Added a comment!